internal/client/client.go

```diff
@@ -472,7 +457,7 @@ func (c *CSAPI) MustDoFunc(t *testing.T, method string, paths []string, opts ...
 	if res.StatusCode < 200 || res.StatusCode >= 300 {
 		defer res.Body.Close()
 		body, _ := ioutil.ReadAll(res.Body)
-		t.Fatalf("CSAPI.MustDoFunc response return non-2xx code: %s - body: %s", res.Status, string(body))
+		t.Fatalf("CSAPI.MustDoFunc %s %s returned non-2xx code: %s - body: %s", method, res.Request.URL.String(), res.Status, string(body))
```

Comment:
`MustDo` was a bit more verbose this way, and I remember seeing it useful in the logs the few times that the codebase used `MustDo` over `MustDoFunc`, it's much clearer what exactly complement is doing that way.